addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5330,7 +5339,7 @@ msgstr ""
 
 #: unknown
 msgctxt "#13109"
-msgid "Test & apply resolution"
+msgid "Test & apply the resolution"
```

Comment:
Technically something like this doesn't need "the", but I don't feel strongly about it either way.